File: 251009/짝수만 거꾸로 출력하기/print-even-numbers-upside-down.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;

    int array[a];
    for (int i = 0; i < a; i++) {
        cin >> array[i];
    }

    for(int i = a-1; i >= 0; i--) {
        if(array[i] % 2 == 0) {
            cout << array[i] << " ";
        }
    }
    return 0;
}

File: 251116/짝수만 거꾸로 출력하기/print-even-numbers-upside-down.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());
        int[] save = new int[a];
        string[] input = Console.ReadLine().Split();

        save = Array.ConvertAll(input, int.Parse);

        for(int i = a-1; i >= 0; i--) {
            if(save[i] % 2 == 0) {
                Console.Write(save[i] + " ");
            }
        }
    }
}
